cmd/spark-connect-example-spark-session/main.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 var (
-	remote = flag.String("remote", "localhost:15002",
+	remote = flag.String("remote", "sc://localhost:15002",
```

Comment:
I think this is already fixed, maybe rebase?

Reply:
You are right! Did rebase just now.